internal/pkg/fixtures/configs/from-template.json

```diff
@@ -4,6 +4,12 @@
   "description": "test fixture",
   "changeDescription": "From template keboola/my-template-id/1.2.3",
   "configuration": {
+    "authorization": {
+      "oauth_api": {
+        "id": "1234",
+        "version": 3
+      }
+    },
```

Comment:
Tohle je fixture pro konfig, který se použije v `instance-upgrade-inputs` testu. Je v něm ta autorizace a níž se v `expected-response` podíváme jestli se dostala do defaultu toho inputu.)

(Neřeš teď prosím tě kde má nebo nemá být to `oauth_api`, já to potom sjednotím až se fixne tenhle problém. 😅)